switchapi/es2k/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
-# CMake build file for krnlmon/switchapi/es2k
+#CMake build file for krnlmon / switchapi / es2k
```

Comment:
Minor nit: use space after the initial `#` before text in the comments in this file